code/controllers/subsystems/processing/slow_process.dm

```diff
@@ -0,0 +1,10 @@
+/var/datum/controller/subsystem/processing/slow/SSslow_process
+
+// Not really much to it, literally just a slower-ticking SSprocessing.
+
+/datum/controller/subsystem/processing/slow
+    name = "Processing (Slow)"
+    wait = 60
+
+/datum/controller/subsystem/processing/slow/New()
+    NEW_SS_GLOBAL(SSslow_process)
```

Comment:
Unnecessary for one or few objects. Use the regular SSprocessing and use time delta calculations.

```dm
/some/obj
	var/time_last_ran = 0

/some/obj/proc/process()
	if (world.time - time_last_ran < 60)
		return

	time_last_ran = world.time
```